ee/app/livechat-enterprise/server/api/units.ts

```diff
@@ -0,0 +1,92 @@
+import { API } from '../../../../../app/api/server';
+import { findUnits, findUnitById, findUnitMonitors } from './lib/units';
+import { LivechatEnterprise } from '../lib/LivechatEnterprise';
+import { ILivechatBusinessUnit } from '../../../../../definition/ILivechatBusinessUnit';
+
+API.v1.addRoute('livechat/units.list', { authRequired: true }, {
+	async get() {
+		const { offset, count } = this.getPaginationItems();
+		const { sort } = this.parseJsonQuery();
+		const { text } = this.queryParams;
+
+		const response = await findUnits({
+			userId: this.userId,
+			text,
+			pagination: {
+				offset,
+				count,
+				sort,
+			},
+		});
+
+		return API.v1.success(response);
+	},
+});
+
+API.v1.addRoute('livechat/units.getOne', { authRequired: true }, {
+	async get() {
+		const { id } = this.urlParams;
+		const { unit } = await findUnitById({
+			userId: this.userId,
+			unitId: id,
+		}) as { unit: ILivechatBusinessUnit };
+
+		return API.v1.success(unit);
+	},
+});
+
+API.v1.addRoute('livechat/unitMonitors.list', { authRequired: true }, {
+	async get() {
+		const { unitId } = this.queryParams;
+
+		return API.v1.success(await findUnitMonitors({
+			userId: this.userId,
+			unitId,
+		}));
+	},
+});
+
+API.v1.addRoute('livechat/units', { authRequired: true, permissionsRequired: ['manage-livechat-units'] }, {
+	async get() {
```

Comment:
@KevLehman Y r we creating a new endpoint when we already have one with the same functionality? Talking about this one here - `livechat/units.list` - GET